sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -55,257 +51,220 @@
  * This is meant to be used only internally as a bridge access to classes in
  * com.azure.cosmos
  **/
+@Warning(value = "Internal use only, not meant for public usage")
```

Comment:
you might be able to define a static final warning String message as string and reuse, rather than repeating the warning on each method.

Reply:
Good point, will do that for sure.